guides/common/modules/con_overview-of-image-mode-hosts.adoc

```diff
@@ -0,0 +1,27 @@
+[id="overview-of-image-mode-hosts_{context}"]
+= Overview of image mode hosts
```

Comment:
This heading would work if it was the first section inside a chapter, but it doesn't quite work as a chapter title.

I think that "Package mode and image mode" worked better.